paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -32,12 +32,17 @@
         "executor": {"enum": ["ssh", "paasta"]},
         "cpus": {
           "type": "number",
-          "minumum": 0,
+          "minimum": 0,
```

Comment:
maybe we should get jsonschema validation for our json schemas 😆

Reply:
as well as our json schema validation for that validation